test/tests/test_logging/test_function_logs.sh

```diff
@@ -43,11 +42,18 @@ echo "Grabbing logs, should have 4 calls in logs"
 
 sleep 15
 
-logs=$(fission function logs --name $fn --detail)
+fission function logs --name $fn --detail > logfile
+
+size=$(wc -c <logfile)
+if [ $size = 0 ]
+then
+    fission function logs --name $fn --detail > logfile
```

Comment:
This is the same command as line 45?  What's the goal of doing this again?

Reply:
sometimes the 'fission function logs' command doesn't work the first time around, and it works with a retry of the command.. this is probably due to the time it takes for the logs to be shipped into Influx. The retry is to ensure the test does not fail because of this delay; if the retry fails and the command does not return anything the second time around then we can assume that it does not work properly